cmd/maya-exporter/app/collector/metrics.go

```diff
@@ -260,6 +261,14 @@ func Metrics(cas string) metrics {
 				Help:      "iSCSI Initiator to jiva target login status: (0, 1): {Not Logged In, Logged In}",
 			},
 		),
+
+		targetRejectRequestCounter: prometheus.NewCounter(
+			prometheus.CounterOpts{
+				Namespace: "openebs",
+				Name:      "target_reject_request_counter",
+				Help:      "Total no of rejected GET http requests if a request is already in progress",
```

Comment:
n_: `Total number of rejected scrape requests if a previous request is in progress`